app/components/containerConsole/containerConsole.html

```diff
@@ -33,8 +33,13 @@
                 </select>
               </div>
             </div>
+            <div class="col-sm-3">
+              <div class="input-group">
+                <input class="form-control" type="text" name="user" ng-model="state.user" placeholder="User">
+              </div>
+            </div>
             <!-- !command-list -->
-            <div class="col-sm-8">
+            <div class="col-sm-5">
```

Comment:
Well, either change the text to:

* Format is one of: user, user:group, uid or uid:gid
* Make command and user column wider